modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/freelist/PagesList.java

```diff
@@ -866,7 +862,7 @@ protected final void put(
 
                 assert getPageId(tailAddr) == tailId
                         : "tailId = " + hexLong(tailId) + ", pageId = " + hexLong(getPageId(tailAddr));
-                assert getType(tailAddr) == T_PAGE_LIST_NODE
+                assert getType(tailAddr) == PagesListNodeIo.VERSIONS.getType()
```

Comment:
There can be a constant for this in `PagesListNodeIo`, it's fine

Reply:
Peplace it